tensorflow_serving/batching/batching_session.cc

```diff
@@ -20,6 +20,8 @@ limitations under the License.
 #include "tensorflow/core/framework/tensor.h"
```

Comment:
Naturally we'll want (1) documentation in the header file, and (2) test coverage. (I gather you want feedback on the basic approach before going all the way with those aspects, which makes sense.)